### ISSUE TITLE
 新作rep分析功能【功能添加意见】 #28 (enhancement)

### ISSUE THREAD

**Issue by toimine** [touhouworldcup/thprac#14]
[Kerbinator-CN](https://github.com/Kerbinator-CN) commented [on 9 May 2021](https://github.com/ack7139/thprac/issues/28#issue-882483663)
比较新的新作似乎没有红妖永的那个统计功能
就是红妖永"你的战绩"部分
希望能添加一个功能，在replay播放（包括加速播放）的时候能够统计：
miss次数，bomb次数，SCB次数
飞碟/灵界/季节解放/咆哮等的次数
抱掉的B总数，决死数等等
并且在rep播放结束的时候显示战绩
————————————————
其实意思就是：很新的游戏似乎完全没存储这些数据，需要rep通过游戏你计算 （rep炸了另说）
而且如果说通过弹幕还原那至少需要游戏内高速播放一次rep

**Comment by H-J-Granger** [comment]
English Translation:

Title: Replay analysis feature for recent games [enhancement]

[Kerbinator-CN](https://github.com/Kerbinator-CN) commented [on 9 May 2021](https://github.com/ack7139/thprac/issues/28#issue-882483663) 

It seems that the newer Touhou games no longer have the statistics feature from EoSD, PCB, and IN—specifically, the 'Your Performance' section.

I hope a feature can be added to count the following during replay playback (including fast-forwarded playback):

- Number of misses, bombs, and spell card bonuses (SCBs)
- Counts of UFO summons, Trance activations, Season releases, Roaring activations, etc.
- Total number of bombs wasted, deathbombs, and so on
- And display the statistics at the end of the replay.

------

In short: The very recent games don’t seem to store these statistics at all, so they have to be calculated dynamically from the replay (assuming not desync-ed).
And if you were to reconstruct the data from danmaku, at the very least, the replay would need to be played back at high speed in-game.

**Comment by 32th-System** [comment]
Does the ingame tracker existing mean that this is technically already implemented?
